packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -254,6 +262,16 @@ const MessageThreadStory = (args): JSX.Element => {
   const [chatMessages, setChatMessages] = useState<(SystemMessage | CustomMessage | ChatMessage)[]>(
     GenerateMockChatMessages()
   );
+  const dropdownMenuOptions = [
+    { key: 'newMessage', text: 'New Message' },
```

Comment:
```suggestion
    { key: 'newMessage', text: 'New Message from you' },
```

Reply:
I'd like to stick with the original text since we've received no feedback suggesting it's confusing to users. As it's the default option for sending a message, it seems quite clear and easy to understand.